### PR TITLE
PAGOPA-726 adding stations creditor institutions filter

### PR DESCRIPTION
**Pull request by FedericoRuzzier**
#### List of Changes
Adding a filter for the stations based on the creditor institution

#### Motivation and Context
This change was required by selfcare to properly filter the stations

#### How Has This Been Tested?
This has been tested manually through postman and through Junit tests

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/main/java/it/gov/pagopa/apiconfig/selfcareintegration/config/ResponseValidator.java [hunk 1]
```diff
@@ -28,7 +28,7 @@ public class ResponseValidator {
    * @param joinPoint not used
    * @param result    the response to validate
    */
-  @AfterReturning(pointcut = "execution(* it.gov.pagopa.apiconfig.selfcareintegration.controller.*.*(..))", returning = "result")
+  @AfterReturning(pointcut = "execution(* it.gov.pagopa.apiconfig.controller.*.*(..))", returning = "result")
```
[the remark this answers is not in the document]
Comment (FedericoRuzzier) [reply src/main/java/it/gov/pagopa/apiconfig/selfcareintegration/config/ResponseValidator.java:31]: I think it was right before, reverting

**Review by jacopocarlini — APPROVED** [review]
👍

File: openapi/generate_openapi.sh [hunk 2]
```diff
@@ -1,20 +1,38 @@
...
+    sleep 5
+done
+echo 'Service Started'
 
-# save openapi
-curl http://localhost:8080/v3/api-docs > ./openapi.json
 
-# UI mode http://localhost:8080/swagger-ui/index.html
+curl http://127.0.0.1:8080/v3/api-docs > ./openapi/openapi.json
+api-spec-converter  --from=openapi_3 --to=swagger_2 ./openapi/openapi.json > ./openapi/swagger.json
```
Comment (jacopocarlini) [inline openapi/generate_openapi.sh:38]: in this case we don't need swagger v2 file.
```suggestion
```

**Review by jacopocarlini — CHANGES_REQUESTED** [review]

**Review by jacopocarlini — APPROVED** [review]
perfect ⭐

**Comment by jacopocarlini** [comment]
update_code

**Pull request merged**